lib/avrolution/compatibility_check.rb

```diff
@@ -0,0 +1,108 @@
+require 'private_attr'
+require 'diffy'
+require 'avromatic'
```

Comment:
It does currently because that is where we monkey-patch the schema registry client. I could probably cut this require down to `require 'avromatic/schema_registry_patch`.

This dependency is another argument for just extracting a separate gem with a schema registry client that supports our extensions -- and I plan to do that soon.